src/main/java/com/clouway/push/server/PushEvent.java

```diff
@@ -3,11 +3,11 @@
 /**
  * @author Ivan Lazov <ivan.lazov@clouway.com>
  */
-public abstract class PushEvent {
```

Comment:
Why remove `abstract` ?
From what I remember in BSS/WSS there are events that extend `PushEvent`.

Reply:
Actually yeah, the class does not need to be abstract.
Keep this change.